docs/modules/ROOT/pages/how-to/day2ops/remove-organization.adoc

```diff
@@ -0,0 +1,26 @@
+= Remove an organization
+
+== Prerequisites
+
+* All namespaces belonging to that organisation, on every APPUiO Cloud zone, are removed.
+* https://kb.vshn.ch/appuio-cloud/how-to/day2ops/connect-control-api.html[Connect to control api]
+
+
+== Check for organization namespaces
+
+On every APPUiO Cloud zone:
+
+[source,bash]
+----
+kubectl get ns -ocustom-columns="NAME:.metadata.name,ORG:.metadata.labels.appuio\.io\/organization" | grep <ORG_NAME>
```

Comment:
```suggestion
kubectl get ns -o name -l appuio.io/organization=<ORG_NAME>
```

There is a simpler and more efficient way to do this. The `-l` option allows you to filter by label (server side, no need for grep or anything like that). With the above, only namespaces of that organization will be printed.

On a side node, the `-L` option will display a label as an additional column in the output. No need for the convoluted `-ocustom-columns`.